integrationtests/toolprovider/toolprovider_test_mise_bitrise.yml

```diff
@@ -15,5 +15,15 @@ workflows:
             - content: |
                 set -ex
                 node --version | grep -q "22"
-                npm install -g yarn
+                npm install -g @expo/cli
```

Comment:
We can't install yarn anymore because it already exists thanks to corepack and this command throws an error.